docs/upgrade.txt

```diff
@@ -89,6 +89,9 @@ This library version introduces the following breaking changes:
   of this behavior, you cannot have two separate ``id`` and ``_id`` fields in your
   documents.
 
+- When using a ``DateTimeInterface`` object in a query, the library converts
+  the ``DateTimeInterface`` to a ``MongoDB\BSON\UTCDateTime`` object.
```

Comment:
S: It's a bit unclear what it means to use an object "in a query". Perhaps you could provide a short example (code or descriptive)

Reply:
Query refers to the filter passed to `Query\Builder::where()`, or the data to `insert()` or `update()`.